ref-exchange/src/errors.rs

```diff
@@ -14,13 +14,17 @@ pub const ERR22_NOT_ENOUGH_TOKENS: &str = "E22: not enough tokens in deposit";
 pub const ERR24_NON_ZERO_TOKEN_BALANCE: &str = "E24: non-zero token balance";
 pub const ERR25_CALLBACK_POST_WITHDRAW_INVALID: &str =
     "E25: expected 1 promise result from withdraw";
-pub const ERR26_ACCESS_KEY_NOT_ALLOWED: &str = "E26: access key not allowed";
+// [AUDIT_05]
+// pub const ERR26_ACCESS_KEY_NOT_ALLOWED: &str = "E26: access key not allowed";
+pub const ERR27_DEPOSIT_NEEDED: &str = 
```

Comment:
I think if we want to be super clear this should be "Attach 1 yN to swap tokens not in whitelist" or something. Deposit is a bit overloaded in this context. There is value in saying that access key is not allowed - because the attached deposit is indeed to prevent access keys to doing this operation.

Reply:
OK